spec/ruby-stackoverflow/response_data_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+require 'json'
+
+module RubyStackoverflow
+  describe Client::ResponseData do
+    it 'should response data fields' do
+      data = JSON.parse(fixture("users.json").read, {:symbolize_names => true})
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.